api.swagger.yaml

```diff
@@ -10862,6 +11037,187 @@ components:
+                    type: string
+                    format: uuid
+                    example: '01234567-89ab-cdef-1234-56789abcdef0'
+                  name:
+                    description: 'The name of the idea category.'
+                    type: string
+                    example: 'Important'
+            author:
+              description: 'The author of this idea.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/user-trimmed-schema'
+            timestamp_created:
+              description: 'This is the timestamp (in seconds) of when the idea was created.'
+              type: integer
+              example: 1510941596
+            timestamp_updated:
+              description: 'This is the timestamp (in seconds) of when the idea was last updated.'
+              type: integer
+              example: 1510941696
+            comment_count:
+              description: 'Count of comments on the idea.'
+              type: integer
+              example: 5
+            likes_count:
+              description: 'Count of likes on the idea.'
+              type: integer
+              example: 5
+            published:
+              description: 'Published status for the idea.'
+              type: boolean
+              example: true
+    idea-schema-update:
+      allOf:
+        - $ref: '#/components/schemas/idea-schema-common'
+        - type: object
+          properties:
+            idea_status:
+              description: 'The idea status.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the idea status.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            idea_assignee:
+              description: 'The assignee of this idea.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the user.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            idea_categories:
+              description: 'List of categories associated with the idea.'
+              type: array
+              items:
+                type: object
+                properties:
+                  uuid:
+                    description: 'Invotra UUID of the idea category.'
+                    type: string
+                    format: uuid
+                    example: '01234567-89ab-cdef-1234-56789abcdef0'
+            author:
+              description: 'The author of this idea.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the user.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+    idea-schema-create:
```

Comment:
These properties are not set during create by end users. End users fill out a webform which generates an idea, administrators can then edit these fields.

We'll have to tie this in with the webform APIs